mobile/src/reducers/calendar/event-dialog/event-dialog.epics.ts

```diff
@@ -59,8 +59,9 @@ export const openEventDialogEpic$ = (action$: ActionsObservable<EventDialogActio
 
 export const closeEventDialogEpic$ = (action$: ActionsObservable<EventDialogActions>) =>
     action$.ofType('CLOSE-EVENT-DIALOG')
-        .flatMap(x => Observable.of<CalendarActions>(
+        .flatMap(x => Observable.of<CalendarActions | EventDialogActions>(
             calendarSelectionMode(CalendarSelectionModeType.SingleDay),
             disableSelectIntervalsBySingleDaySelection(false),
-            selectIntervalsBySingleDaySelection()
+            selectIntervalsBySingleDaySelection(),
+            stopProgress()
```

Comment:
if you stop event processing here, you need to start it in epics as well. Just map all 'confirm' action to one action instead of dispatching it from components

Reply:
I wanted to do it synchronously to disable the buttons immediately

Reply:
fixed